packages/dd-trace/src/debugger/devtools_client/condition.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+
+module.exports = compile
+
+function compile (node) {
+  if (node === null || typeof node === 'number' || typeof node === 'boolean' || typeof node === 'string') {
+    return JSON.stringify(node)
+  }
+
+  const [type, value] = Object.entries(node)[0]
+
+  if (type === 'not') {
+    return `!(${compile(value)})`
+  } else if (type === 'isEmpty') {
+    return `(${compile(value)}).length === 0`
+  } else if (type === 'isDefined') {
+    return `(() => {
+        try {
+          ${value}
+          return true
+        } catch {
+          return false
+        }
+      })()`
+  } else if (type === 'instanceof') {
+    return `${compile(value[0])} instanceof ${value[1]}`
```

Comment:
should this be under Array.isArray(value) at line 39 ?

Reply:
Is it because `value[1]` should also be compiled?

Reply:
nope. because you access it as an array...

Reply:
Under the `Array.isArray(value)` if-block below, I also compile the second element of the `value` array. If we don't want that in this case, it shouldn't be part of that if-block